enterprise-cfengine-guide/settings.markdown

```diff
@@ -196,6 +196,33 @@ Mission portal can authenticate against an external directory.
 
 **See Also:** [LDAP authentication REST API][LDAP authentication API]
 
+
+## Role based access control ##
+
+![Role based access control](role_based_access_control_settings.png)
+
+Roles in Mission portal can be restricted to perform only configured actions. 
+And you can configure they on Role based access control settings page.
```

Comment:
s/And y/Y/

Reply:
Actually, second sentence: Configure role-based access controls from settings.

Reply:
done, thanks